specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/CommonDefinitions.json

```diff
@@ -1614,6 +1614,208 @@
         }
       }
     },
+    "ManagedHttpRoutePatch": {
+      "description": "A managed http route to update",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Application-specific metadata in the form of key-value pairs."
+        }
+      }
+    },
+    "ManagedHttpRoute": {
+      "description": "Managed certificates used for Custom Domain bindings of Container Apps in a Managed Environment",
```

Comment:
These several of these descriptions look like they were copied from another type? Please update them.

Reply:
fixed